src/test/common/test_json_formatter.cc

```diff
@@ -23,14 +24,12 @@
 
 using namespace std;
 
+const string outstring = 
```

Comment:
I'm surprised you didn't make this a raw string literal. Not a complain, just surprised.